src/MyMap/MyHashMap.java

```diff
@@ -41,6 +41,7 @@ private final int getHashCode(Object key) {
             hash = hash * ASCIISymCnt + sKeyString.charAt(i);
         }
 
+        if (hash < 0) hash *= -1;
```

Comment:
Can we have an "unsigned int" for not to have such a problem?

Reply:
Java does not have a datatype for unsigned integers.
We can define a long instead of an int, but even with long I had a negative value problem.